betterforms/changelist.py

```diff
@@ -101,7 +101,7 @@ def get_queryset(self):
         return qs
 
 
-class BoundHeader(object):
+class BoundHeader():
```

Comment:
This and all other base classes that implicitly subclass `object` should not have parens. (`class BoundHeader:`).